include/fc/io/cfile.hpp

```diff
@@ -53,10 +53,25 @@ class cfile {
       if( !_file ) {
          throw std::ios_base::failure( "cfile unable to open: " +  _file_path.generic_string() + " in mode: " + std::string( mode ) );
       }
+
+
       _open = true;
    }
 
+   bool remaining(){
+      if(!is_open() || eof())
+         return false;
+
+      int c(getc());
+      bool isEOF(eof());
```

Comment:
Really don't like this syntax. Can you at least use `{}` instead of `()` for both of these.

Reply:
no problem. I like to use the initialization list even though it is non-traditional. I will just use the traditional syntax..